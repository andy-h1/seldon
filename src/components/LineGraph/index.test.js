import React from 'react';
import { render, waitFor } from '@testing-library/react';
import { LineGraph } from '.';

describe('Line Graph component', () => {
  it('should render', async () => {
    const { container } = render(<LineGraph />);
    await waitFor(() => expect(container.firstChild).toMatchSnapshot());
  });
});
