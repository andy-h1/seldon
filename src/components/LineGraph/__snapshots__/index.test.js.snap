// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Line Graph component should render 1`] = `
<svg
  height="300"
  width="950"
>
  <g
    transform="translate(50, 50)"
  >
    <text
      class="tooltip"
    />
    <text
      class="tooltipDay"
    />
    <g
      fill="none"
      font-family="sans-serif"
      font-size="10"
      text-anchor="middle"
      transform="translate(0, 200)"
    >
      <path
        class="domain"
        d="M0.5,6V0.5H850.5V6"
        stroke="currentColor"
      />
      <g
        class="tick"
        opacity="1"
        transform="translate(0.5,0)"
      >
        <line
          stroke="currentColor"
          y2="6"
        />
        <text
          dy="0.71em"
          fill="currentColor"
          y="9"
        >
          0
        </text>
      </g>
      <g
        class="tick"
        opacity="1"
        transform="translate(85.5,0)"
      >
        <line
          stroke="currentColor"
          y2="6"
        />
        <text
          dy="0.71em"
          fill="currentColor"
          y="9"
        >
          2
        </text>
      </g>
      <g
        class="tick"
        opacity="1"
        transform="translate(170.5,0)"
      >
        <line
          stroke="currentColor"
          y2="6"
        />
        <text
          dy="0.71em"
          fill="currentColor"
          y="9"
        >
          4
        </text>
      </g>
      <g
        class="tick"
        opacity="1"
        transform="translate(255.5,0)"
      >
        <line
          stroke="currentColor"
          y2="6"
        />
        <text
          dy="0.71em"
          fill="currentColor"
          y="9"
        >
          6
        </text>
      </g>
      <g
        class="tick"
        opacity="1"
        transform="translate(340.5,0)"
      >
        <line
          stroke="currentColor"
          y2="6"
        />
        <text
          dy="0.71em"
          fill="currentColor"
          y="9"
        >
          8
        </text>
      </g>
      <g
        class="tick"
        opacity="1"
        transform="translate(425.5,0)"
      >
        <line
          stroke="currentColor"
          y2="6"
        />
        <text
          dy="0.71em"
          fill="currentColor"
          y="9"
        >
          10
        </text>
      </g>
      <g
        class="tick"
        opacity="1"
        transform="translate(510.5,0)"
      >
        <line
          stroke="currentColor"
          y2="6"
        />
        <text
          dy="0.71em"
          fill="currentColor"
          y="9"
        >
          12
        </text>
      </g>
      <g
        class="tick"
        opacity="1"
        transform="translate(595.5,0)"
      >
        <line
          stroke="currentColor"
          y2="6"
        />
        <text
          dy="0.71em"
          fill="currentColor"
          y="9"
        >
          14
        </text>
      </g>
      <g
        class="tick"
        opacity="1"
        transform="translate(680.5,0)"
      >
        <line
          stroke="currentColor"
          y2="6"
        />
        <text
          dy="0.71em"
          fill="currentColor"
          y="9"
        >
          16
        </text>
      </g>
      <g
        class="tick"
        opacity="1"
        transform="translate(765.5,0)"
      >
        <line
          stroke="currentColor"
          y2="6"
        />
        <text
          dy="0.71em"
          fill="currentColor"
          y="9"
        >
          18
        </text>
      </g>
      <g
        class="tick"
        opacity="1"
        transform="translate(850.5,0)"
      >
        <line
          stroke="currentColor"
          y2="6"
        />
        <text
          dy="0.71em"
          fill="currentColor"
          y="9"
        >
          20
        </text>
      </g>
    </g>
    <g
      class="grid"
      fill="none"
      font-family="sans-serif"
      font-size="10"
      text-anchor="end"
    >
      <path
        class="domain"
        d="M850,200.5H0.5V0.5H850"
        stroke="currentColor"
      />
      <g
        class="tick"
        opacity="1"
        transform="translate(0,200.5)"
      >
        <line
          stroke="currentColor"
          x2="850"
        />
        <text
          dy="0.32em"
          fill="currentColor"
          x="-3"
        />
      </g>
      <g
        class="tick"
        opacity="1"
        transform="translate(0,180.5)"
      >
        <line
          stroke="currentColor"
          x2="850"
        />
        <text
          dy="0.32em"
          fill="currentColor"
          x="-3"
        />
      </g>
      <g
        class="tick"
        opacity="1"
        transform="translate(0,160.5)"
      >
        <line
          stroke="currentColor"
          x2="850"
        />
        <text
          dy="0.32em"
          fill="currentColor"
          x="-3"
        />
      </g>
      <g
        class="tick"
        opacity="1"
        transform="translate(0,140.5)"
      >
        <line
          stroke="currentColor"
          x2="850"
        />
        <text
          dy="0.32em"
          fill="currentColor"
          x="-3"
        />
      </g>
      <g
        class="tick"
        opacity="1"
        transform="translate(0,120.5)"
      >
        <line
          stroke="currentColor"
          x2="850"
        />
        <text
          dy="0.32em"
          fill="currentColor"
          x="-3"
        />
      </g>
      <g
        class="tick"
        opacity="1"
        transform="translate(0,100.5)"
      >
        <line
          stroke="currentColor"
          x2="850"
        />
        <text
          dy="0.32em"
          fill="currentColor"
          x="-3"
        />
      </g>
      <g
        class="tick"
        opacity="1"
        transform="translate(0,80.5)"
      >
        <line
          stroke="currentColor"
          x2="850"
        />
        <text
          dy="0.32em"
          fill="currentColor"
          x="-3"
        />
      </g>
      <g
        class="tick"
        opacity="1"
        transform="translate(0,60.50000000000001)"
      >
        <line
          stroke="currentColor"
          x2="850"
        />
        <text
          dy="0.32em"
          fill="currentColor"
          x="-3"
        />
      </g>
      <g
        class="tick"
        opacity="1"
        transform="translate(0,40.49999999999999)"
      >
        <line
          stroke="currentColor"
          x2="850"
        />
        <text
          dy="0.32em"
          fill="currentColor"
          x="-3"
        />
      </g>
      <g
        class="tick"
        opacity="1"
        transform="translate(0,20.499999999999996)"
      >
        <line
          stroke="currentColor"
          x2="850"
        />
        <text
          dy="0.32em"
          fill="currentColor"
          x="-3"
        />
      </g>
      <g
        class="tick"
        opacity="1"
        transform="translate(0,0.5)"
      >
        <line
          stroke="currentColor"
          x2="850"
        />
        <text
          dy="0.32em"
          fill="currentColor"
          x="-3"
        />
      </g>
    </g>
    <g
      fill="none"
      font-family="sans-serif"
      font-size="10"
      text-anchor="end"
    >
      <path
        class="domain"
        d="M-6,200.5H0.5V0.5H-6"
        stroke="currentColor"
      />
      <g
        class="tick"
        opacity="1"
        transform="translate(0,200.5)"
      >
        <line
          stroke="currentColor"
          x2="-6"
        />
        <text
          dy="0.32em"
          fill="currentColor"
          x="-9"
        >
          0.0
        </text>
      </g>
      <g
        class="tick"
        opacity="1"
        transform="translate(0,180.5)"
      >
        <line
          stroke="currentColor"
          x2="-6"
        />
        <text
          dy="0.32em"
          fill="currentColor"
          x="-9"
        >
          0.1
        </text>
      </g>
      <g
        class="tick"
        opacity="1"
        transform="translate(0,160.5)"
      >
        <line
          stroke="currentColor"
          x2="-6"
        />
        <text
          dy="0.32em"
          fill="currentColor"
          x="-9"
        >
          0.2
        </text>
      </g>
      <g
        class="tick"
        opacity="1"
        transform="translate(0,140.5)"
      >
        <line
          stroke="currentColor"
          x2="-6"
        />
        <text
          dy="0.32em"
          fill="currentColor"
          x="-9"
        >
          0.3
        </text>
      </g>
      <g
        class="tick"
        opacity="1"
        transform="translate(0,120.5)"
      >
        <line
          stroke="currentColor"
          x2="-6"
        />
        <text
          dy="0.32em"
          fill="currentColor"
          x="-9"
        >
          0.4
        </text>
      </g>
      <g
        class="tick"
        opacity="1"
        transform="translate(0,100.5)"
      >
        <line
          stroke="currentColor"
          x2="-6"
        />
        <text
          dy="0.32em"
          fill="currentColor"
          x="-9"
        >
          0.5
        </text>
      </g>
      <g
        class="tick"
        opacity="1"
        transform="translate(0,80.5)"
      >
        <line
          stroke="currentColor"
          x2="-6"
        />
        <text
          dy="0.32em"
          fill="currentColor"
          x="-9"
        >
          0.6
        </text>
      </g>
      <g
        class="tick"
        opacity="1"
        transform="translate(0,60.50000000000001)"
      >
        <line
          stroke="currentColor"
          x2="-6"
        />
        <text
          dy="0.32em"
          fill="currentColor"
          x="-9"
        >
          0.7
        </text>
      </g>
      <g
        class="tick"
        opacity="1"
        transform="translate(0,40.49999999999999)"
      >
        <line
          stroke="currentColor"
          x2="-6"
        />
        <text
          dy="0.32em"
          fill="currentColor"
          x="-9"
        >
          0.8
        </text>
      </g>
      <g
        class="tick"
        opacity="1"
        transform="translate(0,20.499999999999996)"
      >
        <line
          stroke="currentColor"
          x2="-6"
        />
        <text
          dy="0.32em"
          fill="currentColor"
          x="-9"
        >
          0.9
        </text>
      </g>
      <g
        class="tick"
        opacity="1"
        transform="translate(0,0.5)"
      >
        <line
          stroke="currentColor"
          x2="-6"
        />
        <text
          dy="0.32em"
          fill="currentColor"
          x="-9"
        >
          1.0
        </text>
      </g>
    </g>
    <path
      class="area"
      d="M0,55.48646368388888C14.166666666666666,127.25621116666886,28.333333333333336,199.02595864944885,42.5,199.02595864944885C56.666666666666664,199.02595864944885,70.83333333333333,134.18842425313602,85,134.18842425313602C99.16666666666667,134.18842425313602,113.33333333333333,186.10029099511297,127.5,186.10029099511297C141.66666666666666,186.10029099511297,155.83333333333334,162.41766144392236,170,162.41766144392236C184.16666666666666,162.41766144392236,198.33333333333334,185.63464932222854,212.5,185.63464932222854C226.66666666666666,185.63464932222854,240.83333333333334,141.3610128610603,255,115.80991385669934C269.1666666666667,90.25881485233839,283.3333333333333,32.328055296062885,297.5,32.328055296062885C311.6666666666667,32.328055296062885,325.8333333333333,187.17434920540865,340,187.17434920540865C354.1666666666667,187.17434920540865,368.3333333333333,158.96511571199306,382.5,144.2069674093907C396.6666666666667,129.44881910678833,410.8333333333333,116.41619140557094,425,98.62545938979449C439.1666666666667,80.834727374018,453.33333333333337,37.46257531473185,467.50000000000006,37.46257531473185C481.6666666666667,37.46257531473185,495.83333333333337,110.25935108693612,510,123.38007704110268C524.1666666666666,136.50080299526925,538.3333333333334,143.06116597235254,552.5,143.06116597235254C566.6666666666666,143.06116597235254,580.8333333333334,139.3675273238797,595,131.98025002693404C609.1666666666666,124.59297272998838,623.3333333333334,11.027191951834837,637.5,11.027191951834837C651.6666666666666,11.027191951834837,665.8333333333334,104.79355736526338,680,108.34512010633506C694.1666666666666,111.89668284740674,708.3333333333334,113.67246421794258,722.5,113.67246421794258C736.6666666666666,113.67246421794258,750.8333333333334,46.180262706139615,765,46.180262706139615C779.1666666666666,46.180262706139615,793.3333333333334,86.95313887468454,807.5,101.38637084272139C821.6666666666666,115.81960281075824,835.8333333333334,124.29962866255948,850,132.77965451436071L850,200C835.8333333333334,200,821.6666666666666,200,807.5,200C793.3333333333334,200,779.1666666666666,200,765,200C750.8333333333334,200,736.6666666666666,200,722.5,200C708.3333333333334,200,694.1666666666666,200,680,200C665.8333333333334,200,651.6666666666666,200,637.5,200C623.3333333333334,200,609.1666666666666,200,595,200C580.8333333333334,200,566.6666666666666,200,552.5,200C538.3333333333334,200,524.1666666666666,200,510,200C495.83333333333337,200,481.6666666666667,200,467.50000000000006,200C453.33333333333337,200,439.1666666666667,200,425,200C410.8333333333333,200,396.6666666666667,200,382.5,200C368.3333333333333,200,354.1666666666667,200,340,200C325.8333333333333,200,311.6666666666667,200,297.5,200C283.3333333333333,200,269.1666666666667,200,255,200C240.83333333333334,200,226.66666666666666,200,212.5,200C198.33333333333334,200,184.16666666666666,200,170,200C155.83333333333334,200,141.66666666666666,200,127.5,200C113.33333333333333,200,99.16666666666667,200,85,200C70.83333333333333,200,56.666666666666664,200,42.5,200C28.333333333333336,200,14.166666666666666,200,0,200Z"
    />
    <path
      class="line"
      d="M0,55.48646368388888C14.166666666666666,127.25621116666886,28.333333333333336,199.02595864944885,42.5,199.02595864944885C56.666666666666664,199.02595864944885,70.83333333333333,134.18842425313602,85,134.18842425313602C99.16666666666667,134.18842425313602,113.33333333333333,186.10029099511297,127.5,186.10029099511297C141.66666666666666,186.10029099511297,155.83333333333334,162.41766144392236,170,162.41766144392236C184.16666666666666,162.41766144392236,198.33333333333334,185.63464932222854,212.5,185.63464932222854C226.66666666666666,185.63464932222854,240.83333333333334,141.3610128610603,255,115.80991385669934C269.1666666666667,90.25881485233839,283.3333333333333,32.328055296062885,297.5,32.328055296062885C311.6666666666667,32.328055296062885,325.8333333333333,187.17434920540865,340,187.17434920540865C354.1666666666667,187.17434920540865,368.3333333333333,158.96511571199306,382.5,144.2069674093907C396.6666666666667,129.44881910678833,410.8333333333333,116.41619140557094,425,98.62545938979449C439.1666666666667,80.834727374018,453.33333333333337,37.46257531473185,467.50000000000006,37.46257531473185C481.6666666666667,37.46257531473185,495.83333333333337,110.25935108693612,510,123.38007704110268C524.1666666666666,136.50080299526925,538.3333333333334,143.06116597235254,552.5,143.06116597235254C566.6666666666666,143.06116597235254,580.8333333333334,139.3675273238797,595,131.98025002693404C609.1666666666666,124.59297272998838,623.3333333333334,11.027191951834837,637.5,11.027191951834837C651.6666666666666,11.027191951834837,665.8333333333334,104.79355736526338,680,108.34512010633506C694.1666666666666,111.89668284740674,708.3333333333334,113.67246421794258,722.5,113.67246421794258C736.6666666666666,113.67246421794258,750.8333333333334,46.180262706139615,765,46.180262706139615C779.1666666666666,46.180262706139615,793.3333333333334,86.95313887468454,807.5,101.38637084272139C821.6666666666666,115.81960281075824,835.8333333333334,124.29962866255948,850,132.77965451436071"
    />
    <circle
      class="dot"
      cx="0"
      cy="55.48646368388888"
      r="5"
    />
    <circle
      class="dot"
      cx="42.5"
      cy="199.02595864944885"
      r="5"
    />
    <circle
      class="dot"
      cx="85"
      cy="134.18842425313602"
      r="5"
    />
    <circle
      class="dot"
      cx="127.5"
      cy="186.10029099511297"
      r="5"
    />
    <circle
      class="dot"
      cx="170"
      cy="162.41766144392236"
      r="5"
    />
    <circle
      class="dot"
      cx="212.5"
      cy="185.63464932222854"
      r="5"
    />
    <circle
      class="dot"
      cx="255"
      cy="115.80991385669934"
      r="5"
    />
    <circle
      class="dot"
      cx="297.5"
      cy="32.328055296062885"
      r="5"
    />
    <circle
      class="dot"
      cx="340"
      cy="187.17434920540865"
      r="5"
    />
    <circle
      class="dot"
      cx="382.5"
      cy="144.2069674093907"
      r="5"
    />
    <circle
      class="dot"
      cx="425"
      cy="98.62545938979449"
      r="5"
    />
    <circle
      class="dot"
      cx="467.50000000000006"
      cy="37.46257531473185"
      r="5"
    />
    <circle
      class="dot"
      cx="510"
      cy="123.38007704110268"
      r="5"
    />
    <circle
      class="dot"
      cx="552.5"
      cy="143.06116597235254"
      r="5"
    />
    <circle
      class="dot"
      cx="595"
      cy="131.98025002693404"
      r="5"
    />
    <circle
      class="dot"
      cx="637.5"
      cy="11.027191951834837"
      r="5"
    />
    <circle
      class="dot"
      cx="680"
      cy="108.34512010633506"
      r="5"
    />
    <circle
      class="dot"
      cx="722.5"
      cy="113.67246421794258"
      r="5"
    />
    <circle
      class="dot"
      cx="765"
      cy="46.180262706139615"
      r="5"
    />
    <circle
      class="dot"
      cx="807.5"
      cy="101.38637084272139"
      r="5"
    />
    <circle
      class="dot"
      cx="850"
      cy="132.77965451436071"
      r="5"
    />
  </g>
</svg>
`;
